libnetwork/cni/config.go

```diff
@@ -133,9 +132,9 @@ func (n *cniNetwork) NetworkRemove(nameOrID string) error {
 
 	// Remove the bridge network interface on the host.
 	if network.libpodNet.Driver == types.BridgeNetworkDriver {
-		link, err := netlink.LinkByName(network.libpodNet.NetworkInterface)
+		link, err := linkByName(network.libpodNet.NetworkInterface)
```

Comment:
Would this work in freebsd if its pointing to a NOOP function. Wouldn't interface still linger around.

Reply:
At the point I wrote this, I wasn't sure exactly what was being removed and why. Is this cleaning up the host-side bridge? If so, I can implement (horribly) by running 'ifconfig bridgename delete'.

This is how I'm doing all the CNI configuration right now because I'm trying to avoid porting netlink...

Reply:
This part would probably be more readable if I move the whole 'linkByName && linkDel' section into config_linux.go. What do you think?

Reply:
The goal here is to remove the bridge interface when you call podman network rm. This is needed because cni will leak this interface. You should definitely not error on bsd, if you cannot remove the bridge for some reason you can ignore this instead.

Reply:
yes, moving it into config_linux.go would be cleaner.